update docs/install.md (#1888)
openshift/hive: docs/install.md

Hive is published to [OperatorHub](https://operatorhub.io/operator/hive-operator) weekly and this is the best method to install and use Hive if you do not need to build from source.

>   1. Create a 'hive' namespace doing
>   `$ oc new-project hive`

Reviewer: Not sure if this indentation is correct but wrap the command in triple ticks ``` please.

```suggestion
  1. Create a 'hive' namespace,
      ```
      oc new-project hive
      ```
```

Author: Done.

>   2. Install the Hive Operator:

Reviewer: These bullets should be prefixed with `1.` s.t. the number enumeration is done by markdown. The newline preceding this line will need to be removed for this to work.

Author: Done. Thanks!

>   * In the [OpenShift web console](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html), navigate to [Administrator perspective](https://docs.openshift.com/container-platform/4.11/web_console/web-console-overview.html#accessing-the-administrator-perspective_web-console-overview) > Operators > [OperatorHub](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-understanding-operatorhub.html#olm-operatorhub-overview_olm-understanding-operatorhub).

Reviewer: I think linking to documentation is okay and we do this in multiple places already (`grep -ir "docs.openshift.com" | wc -l` => `11`) but the links are instantly outdated as soon as a new version is released. The docs site supports specifying `latest` as the version in the URL so perhaps that is a good trade off assuming the URL path remains active.

Author: I wasn't sure how to refer to "latest" in the link. Done. Thanks!

  * Search for “hive” and select the "Hive for Red Hat OpenShift" operator and click Install.
  * Select the “alpha” update channel, install to a specific namespace (select the “hive” namespace previously created), approval strategy: automatic, and press Install.
  * You should now have a hive-operator pod running in the hive namespace.
  3. Create a HiveConfig to trigger the the actual deployment of Hive.

>   * create a hive_config.yaml file with this content:

Reviewer: ```suggestion
  * Create a hive_config.yaml file with the following content,
```

  logLevel: debug
  targetNamespace: hive
```

>   * apply the yaml file doing `$ oc apply -f hive_config.yaml`

Reviewer: Wrap command in ```, indentation may be incorrect.

```suggestion
  * Apply hive_config.yaml,
      ```
      oc apply -f hive_config.yaml
      ```
```

Author: Done. Thanks!
